otherlibs/unix/getgr.c

```diff
@@ -28,7 +28,7 @@ static value alloc_group_entry(struct group *entry)
 
   Begin_roots3 (name, pass, mem);
     name = caml_copy_string(entry->gr_name);
-    pass = caml_copy_string(entry->gr_passwd);
+    pass = caml_copy_string(entry->gr_passwd ? entry->gr_passwd : "");
```

Comment:
could you comment on why this is necessary? (Maybe even a comment in the source?)